src/core/renderers/webgl/managers/FilterManager.js

```diff
@@ -362,6 +364,11 @@ export default class FilterManager extends WebGLManager
         // TODO Caching layer..
         for (const i in uniformData)
         {
+            if (hasOwnProperty.call(shader.uniforms, i))
```

Comment:
Would recommend: `if (shader.uniforms[i] === undefined)` instead, since this is "hot" code and would like to not add in a `call`